clang/lib/Sema/AnalysisBasedWarnings.cpp

```diff
@@ -2633,7 +2646,7 @@ void clang::sema::AnalysisBasedWarnings::IssueWarnings(
                        SourceLocation()) ||
       (!Diags.isIgnored(diag::warn_unsafe_buffer_libc_call, SourceLocation()) &&
        S.getLangOpts().CPlusPlus /* only warn about libc calls in C++ */)) {
-    CallableVisitor(CallAnalyzers).TraverseTranslationUnitDecl(TU);
+    CallableVisitor(CallAnalyzers, TU->getOwningModuleID()).TraverseTranslationUnitDecl(TU);
```

Comment:
ModuleID is implementation detailed that users shouldn't use. Let's use owning module if possible.

Reply:
There's no `operator==` on the `Module` class. Would I be correct in assuming that comparing pointers is the correct way to check for equality (in other words, can I assume that each module only has one associated instance)?

Reply:
Yes

Reply:
Done, looks like it's working.